RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -22,6 +22,18 @@
   overrideTrkQuals      = "tobTecStepSelector:tobTecStep",
 ))
 
+from RecoLocalTracker.SubCollectionProducers.phase2trackClusterRemover_cfi import *
+_convClustersForPhase2PU140 = phase2trackClusterRemover.clone(
+   maxChi2               = cms.double(30.0),
+   trajectories          = cms.InputTag("pixelPairStepTracks"),
+   phase2pixelClusters   = cms.InputTag("siPixelClusters"),
+   phase2OTClusters      = cms.InputTag("siPhase2Clusters"),
+   oldClusterRemovalInfo = cms.InputTag("pixelPairStepClusters"),
+   overrideTrkQuals      = cms.InputTag('pixelPairStepSelector','pixelPairStep'),
+   TrackQuality          = cms.string('highPurity'),
+)
+eras.trackingPhase2PU140.toReplaceWith(convClusters, _convClustersForPhase2PU140)
```

Comment:
I'd avoid explicit types (to enable typo checking) and temporary variable (`_convClustersForPhase2PU140`), and import `from ..._cfi import phase2trackClusterRemover as _phase2trackClusterRemover` also here.